concordia/templates/transcriptions/topic_detail.html

```diff
@@ -30,8 +30,12 @@ <h1>{{ topic.title }}</h1>
                     <h4>Related Links</h4>
                     <div class="list-group">
                         {% for resource in topic.resource_set.all %}
-                            <a class="list-group-item" href="{{ resource.resource_url }}" target="_blank">{{ resource.title }} <i class="fa fa-external-link-alt"></i></a>
-                        {% endfor %}
+                            {% if 'loc.gov' in resource.resource_url   %}
+                        <a class="list-group-item" href="{{ resource.resource_url }}" target="_blank">{{ resource.title }}</a>
```

Comment:
The original code was indented to match the code structure. This should probably be indented for legibility as well.

Reply:
yes, accepted the merge suggestion.